apps/search/app.prod.Dockerfile

```diff
@@ -0,0 +1,71 @@
+# Source
+# https://turbo.build/repo/docs/handbook/deploying-with-docker#example
+# TODO https://pnpm.io/cli/fetch
+
+FROM node:18 AS base
+
+FROM base AS builder
+#RUN apk add --no-cache libc6-compat
+#RUN apk update
+
+# Set working directory
+WORKDIR /app
+
+RUN yarn global add turbo@1.9.1
+RUN yarn global add pnpm@8.2.0
+
+# Copy entire monorepo
+COPY . .
+
+RUN turbo prune --scope="saleor-app-search" --docker
```

Comment:
this command is removing apps and packages not related to the app we want to build
technically we can use single dockerfile for every app, just passing app name to the dockerfile